packages/drivers/odsp-driver/src/odspUtils.ts

```diff
@@ -195,7 +195,13 @@ export async function fetchAndParseAsJSONHelper<T>(
         };
         return res;
     } catch (e) {
-        throwOdspNetworkError(`Error while parsing fetch response: ${e}`, fetchIncorrectResponse, content);
+        throwOdspNetworkError(
+            "errorWhileParsingFetchResponse",
+            fetchIncorrectResponse,
+            content,
+            undefined,
+            { error: Object(e) },
```

Comment:
@jatgarg - What do you expect this will do?  I don't think this is what we want, we should only pass props with "primitive" values here.  And `error` is already used for the error message so this won't be logged anyway.

Reply:
I was hoping that it will be logged as Data_error. Aren't the object logged maybe after serializing?